packages/shared/styles/components/organisms/SfAccordion.scss

```
@@ -1,13 +1,16 @@
 @import "../../helpers";
 .sf-accordion-item {
+  will-change: height;
+  transform: translateZ(0);
+  backface-visibility: hidden;
```

```suggestion
  will-change: height;
  transform: translateZ(0);
  backface-visibility: hidden;
```
@martaradziszewska can we move it to .sf-expand? WDYT?
``` scss
.sf-expand {
  & > * {
    will-change: height;
    transform: translateZ(0);
    backface-visibility: hidden;
  } 
}```